lib/validic/client.rb

```diff
@@ -66,6 +66,16 @@ def connection
 
     private
 
+    def overwrite_creds(options = {})
+      return self if options.is_a?(String) || options[:access_token].nil? || options[:organization_id].nil?
+      access_token = options.delete(:access_token) || Validic.access_token
+      organization_id = options.delete(:organization_id) || Validic.organization_id
+      return self if access_token == @access_token && organization_id == @organization_id
+      self.access_token = access_token
+      self.organization_id = organization_id
+      reload_config
+    end
```

Comment:
no.. please no..

IMO this adds extra unneeded complexity. also this code is not very Ruby like. If they want to use new creds they can just instantiate a new client.. or we can add more intuitive ways of doing this down the line.